apps/web/src/server/views/applications/case/applications-case-menu.component.njk

```diff
@@ -0,0 +1,34 @@
+{% macro applicationsCaseMenu(params) %}
+	{% set application = params.application %}
+	{% set domainType = params.domainType %}
+	{% set selectedPageType = params.selectedPageType %}
+
+	<aside class="govuk-!-margin-bottom-8 pins-list-menu">
+	<ul class="gov-list pins-list-menu">
+		<li class="govuk-heading-s">{{ writeMenuLink({ pageTitle: 'Overview', pageType: '', selectedPageType: selectedPageType,
+											applicationId: application.id, domainType: domainType }) }}</li>
+		<li class="govuk-heading-s">{{ writeMenuLink({ pageTitle: 'Project information', pageType: 'project-information', selectedPageType: selectedPageType,
+											applicationId: application.id, domainType: domainType }) }}</li>
+		<li class="govuk-heading-s">{{ writeMenuLink({ pageTitle: 'Key dates', pageType: 'key-dates', selectedPageType: selectedPageType,
+											applicationId: application.id, domainType: domainType }) }}</li>
+		<li class="govuk-heading-s">{{ writeMenuLink({ pageTitle: 'Project team', pageType: 'project-team', selectedPageType: selectedPageType,
+											applicationId: application.id, domainType: domainType }) }}</li>
+		<li class="govuk-heading-s">{{ writeMenuLink({ pageTitle: 'Fees', pageType: 'fees', selectedPageType: selectedPageType,
+											applicationId: application.id, domainType: domainType }) }}</li>
+		<li class="govuk-heading-s">{{ writeMenuLink({ pageTitle: 'Project documentation', pageType: 'project-documentation', selectedPageType: selectedPageType,
+											applicationId: application.id, domainType: domainType }) }}</li>
+	</ul>
+	</aside>
+{% endmacro %}
+
+{% macro writeMenuLink(params) %}
+	{% set selectedClass = '' %}
+
+	{% if (params.pageType == params.selectedPageType) or (params.pageType == '' and params.selectedPageType == 'overview' ) %}
+		{% set selectedClass = 'pins-selected-item' %}
+	{% endif %}
+
+	<a class="govuk-link {{ selectedClass }}"
+		href="{{ 'view-application'|url({domainType: params.domainType, applicationId: params.applicationId, step: params.pageType }) }}">
```

Comment:
i guess this is the reason for the domaintype. I've checked it but probably the url filter works even without it